src/cljs/rems/spa.cljs

```diff
@@ -93,6 +94,11 @@
  (fn [db _]
    (:theme db)))
 
+(rf/reg-sub
+ :user-language
+ (fn [db _]
+   (:language (:user-settings db))))
+
```

Comment:
We should rather use the existing sub `:language` (from `user_settings.cljs`). This will not work.